atom/browser/native_window_views.cc

```diff
@@ -674,6 +677,15 @@ bool NativeWindowViews::IsMenuBarVisible() {
   return menu_bar_visible_;
 }
 
+void NativeWindowViews::SetVisibleOnAllWorkspaces(bool visible) {
+  is_visible_on_all_workspaces_ = visible;
+  window_->SetVisibleOnAllWorkspaces(visible);
+}
+
+bool NativeWindowViews::IsVisibleOnAllWorkspaces() {
+  return is_visible_on_all_workspaces_;
```

Comment:
Most desktop environments allow users to set windows to be visible on all workspaces, so we should check whether `_NET_WM_STATE_STICKY` is in `_NET_WM_DESKTOP` on Linux to implement `IsVisibleOnAllWorkspaces`.